applications/GeoMechanicsApplication/custom_utilities/stress_strain_utilities.cpp

```diff
@@ -57,6 +58,55 @@ double StressStrainUtilities::CalculateMeanStress(const Vector& StressVector)
     return CalculateTrace(StressVector) / (StressVector.size() == 3 ? 2.0 : 3.0);
 }
 
+double StressStrainUtilities::CalculateLodeAngle(const Vector& StressVector)
```

Comment:
The Kratos Style Guide prescribes to let this function parameter name start with `r`:
```suggestion
double StressStrainUtilities::CalculateLodeAngle(const Vector& rStressVector)
```

This remark also applies to some other functions in the same file.

Reply:
Done throughout the file ( also for DeformationGradient ).